Added bulk reassignment of workspaces roles (#5958)
Scille/parsec-cloud: docs/userguide/share_data.rst

    Parsec client can work while offline (however only data present locally
    are available), synchronization will occur automatically as soon as the
    connection with the server is established.

Bulk assign workspace roles to users
------------------------------------

> Assigning the same roles to multiple users on multiple workspaces can be a bit annoying. Parsec provides a way to bulk assign the roles of one user to another.

Reviewer: I'd use "tedious" instead of "annoying":
```suggestion
Manually assigning the same workspace roles to multiple users can be a bit tedious and error prone. Parsec provides a way to bulk assign the roles from one user to another.
```
In French:
```
L'assignation manuelle des mêmes rôles d'espace de travail à plusieurs utilisateurs peut s'avérer fastidieuse et source d'erreurs. Parsec propose un moyen d'attribuer en bloc les rôles d'un utilisateur à un autre.
```

1. In the Users page, right click on the user you want to copy the roles from:

.. image:: screens/bulk_assign_roles_start.png
    :align: center
    :alt: Start a bulk assignement

2. Search the user you want to assign the roles to:

.. image:: screens/bulk_assign_roles_select_user.png
    :align: center
    :alt: Select the user to assign roles to

> 3. You're shown a summary of what roles are going to be assigned on each workspace. Click on `Assign roles` to complete the operation.

Reviewer: ```suggestion
3. You're shown a summary of which roles are going to be assigned on each workspace. Click on `Assign roles` to complete the operation.
```

.. image:: screens/bulk_assign_roles_summary.png
    :align: center
    :alt: Bulk assignment summary

.. note::

>     1. You can only assign roles on workspaces where you are an Owner or Manager
>     2. Outsider cannot have a role higher than Contributor
>     3. You cannot assign roles higher than your own. If you have the Manager role in a workspace and you're trying to assign the Owner role, it will be downgraded to Contributor).

Reviewer: ```suggestion
    1. You can only assign roles on a workspace where you have an Owner or Manager role
    2. An Outsider user cannot have a role higher than Contributor
    3. You cannot assign roles higher than your own. If you have the Manager role in a workspace and you're trying to assign the Owner role, it will be downgraded to Contributor.
```

>     3. Assigned roles can be downgraded if your own role on the workspace is not high enough (if you're a Manager and you're trying to assign an Owner role, it will be downgraded to Contributor)

Reviewer: In the spec we've decided it should be downgraded to `Manager`. 
Is there a reason why you've chosen `Contributor`?

Also, I would re-phrase it a little bit:
```suggestion
    3. You cannot assign roles higher than your own. If you have the Manager role in a workspace and you're trying to assign the Owner role, it will be downgraded to Contributor).
```

Author: `Managers` can not set other managers, only readers and contributor. Only Owners can assign other Owners.

>     4. If the user already has a higher role on the workspace than the one you're trying to assign, do nothing

Reviewer: ```suggestion
    4. If the user already has a higher role on the workspace than the one you're trying to assign, no change will be made to its role
```

>     5. The option is only available if you have workspace in common with the source user and you have a role high enough

Reviewer: I would move this to the top (as first point)
```suggestion
    5. The option is only available if you have workspaces in common with the source user and you have a role high enough
```
